pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -47,8 +47,8 @@ void ATA::WritePaddedString(u8* data, int* index, const std::string& value, u32
 void ATA::CreateHDDinfo(u64 sizeSectors)
 {
 	//PS2 is limited to 32bit size HDD (2TB), however,
-	//we don't yet support 48bit, so limit to 24bit size
-	constexpr u32 maxSize = 1 << 24;
+	//we don't yet support 48bit, so limit to 28bit size
+	constexpr u32 maxSize = 1 << 28; // 128Gb
```

Comment:
if it's 28 bits, surely this should be 1 << 27?

Reply:
ah yes, but not 1 << 27 but 1 << 28 - 1

Reply:
max size is when all 27 bits are filled with 1

Reply:
fixed

Reply:
Thanks :)

Reply:
Actually can you bracket it for clarity (and so less chance of error) so it's (1 << 28) - 1

Reply:
done

Reply:
Thanks :)